config/errors.yaml

```diff
@@ -76,4 +76,16 @@
 4206:
   :http_status: 403
   :error_code: 4206
-  :description: 'Product can not be deleted due to unauthorized request'
\ No newline at end of file
+  :description: 'Product can not be deleted due to unauthorized request'
+4207:
+  :http_status: 403
+  :error_code: 4207
+  :description: 'Product cannot be transferred due to unauthorized request'
+4208:
+  :http_status: 422
+  :error_code: 4208
+  :description: 'Product cannot be transferred due to a problem with the target user'
```

Comment:
Here we can be more specific. Like notifying that target user is already the owner of the product.

Reply:
This error is for two cases:
- If the target user is an admin
- If the target user is the owner
Maybe I might change it to two errors!